moped-editor/src/views/projects/projectsListView/components/MapDrawer.js

```diff
@@ -102,9 +107,11 @@ function DrawerContent({
  * See https://mui.com/material-ui/react-drawer/#responsive-drawer
  * See https://mui.com/material-ui/react-drawer/#mini-variant-drawer
  */
-export default React.forwardRef(function MapDrawer({ children, title }, ref) {
+export default React.forwardRef(function MapDrawer(
+  { children, title, open, setOpen },
+  ref
+) {
   /* Control desktop drawer and content visibility */
-  const [open, setOpen] = React.useState(true);
```

Comment:
This state was pulled up one level into `ProjectsListViewMap` so that we can pass it to the map component and toggle the drawer when features are selected on the map.